Feature - New Gridsystem core

mmexvr opened the pull request:
**Describe pull-request**  
_Describe what the pull-request is about_

**Merge this before #160  and #161**
Flex based grid system
- **Added columns**
  - Sm 4 column
  - md 8 column
  - lg 16 column
  - xlg 16 column
  - xxlg 16 column
- **Container for 4 different width of the screen**
- **Row system (like bootstrap)**
- Offsets

**Good to know:** 
The gutter and padding is included in the same calculation, instead of margin + padding, because the flex-basis will be complicated solution. This solution contains padding + padding instead for the gutter and padding on the columns

Nesting of grid is not something we will have ( maybe in the future)

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: scania/corporate-ui#589

**How to test**  
_Add description how to test if possible_
- Added code attached in the bottom
- Test breakpoints based on the figma file
- Test browsers
- Test all sizes( You need to try in correct screen breakpoint for every size of columns)
- Test offsets

**Screenshots**  
_If applicable, add screenshots to help explain_
![image](https://user-images.githubusercontent.com/35451568/98120304-15928a00-1eae-11eb-839e-41a60c562175.png)


**Additional context**  
_Add any other context about the pull-request here._
Internet explorer breaks on the next row, height of the row is not increased as aspected

mmexvr committed:
WIP: grid package

mmexvr committed:
WIP: grid system

mmexvr committed:
WIP: cols and offset added

mmexvr committed:
WIP: changed name of mixins

mmexvr committed:
WIP : fix for gulp

mmexvr committed:
Grid system

- Column
- Rows
- container
- Including sizing and media rules for different screens

mmexvr committed:
Remove debugger in grid file

mmexvr committed:
added node v14 in github actions

mmexvr committed:
removed node v14 test

helloimela commented:
To fix on this PR
1. responsiveness
2. typo in prefix file
3. remove the dev settings
4. discuss with niklas: gutterless class on rows, cols, full width

Next Todo:
1. full width option ?
2. px to rem converter
3. pushed grid
4. auto width column
5. hide rule

mmexvr commented:
To fix on this PR
- [x] responsiveness
- [x] typo in prefix file
- [x] remove the dev settings
- [x]  discuss with niklas: gutterless class on rows, cols, full width 

TODOS:
- [x] full width added 
- [x] auto width column #161

mmexvr committed:
Added full-width container

- Fixed responsiveness
- added base unit 4px
- rename prefix file
- added screensizes

mmexvr committed:
Removed all debugg code

helloimela reviewed (approved):
Great job!
